lib/attachinary/view_helpers.rb

```diff
@@ -49,6 +54,7 @@ def attachinary_file_field_options(model, relation, options={})
       options[:html][:data] ||= {}
       options[:html][:data][:attachinary] = options[:attachinary] || {}
       options[:html][:data][:attachinary][:files] = [model.send(relation)].compact.flatten
+      options[:html][:data][:attachinary][:preload] = options[:preload] || 'none'
```

Comment:
adding `preload` to `[:html][:data][:attachinary]` so we can reference it in the `@options` object in javascript